test/unittest/xmlparser/XMLElementParserTests.cpp

```diff
@@ -125,6 +125,69 @@ TEST_F(XMLParserTests, getXMLLifespanQos)
     EXPECT_EQ(XMLP_ret::XML_ERROR, XMLParserTest::getXMLLifespanQos_wrapper(titleElement, lifespan, ident));
 }
 
+/*
+ * This test checks the proper parsing of an octet vecotr xml element, and negative cases.
+ * 1. Correct parsing of a valid element with decimal numbers.
+ * 2. Correct parsing of a valid element with hexadecimal numbers.
+ * 3. Check an bad element with a wrong separator.
+ * 3. Check an bad element with a wrong number.
+ * 3. Check an bad element with a number too high.
+ * 4. Check an  empty xml definition.
```

Comment:
I would add tests mixing decimal and hexadecimal numbers.

`0x20,20,0x10,10` and `10,0x10,20,0x20`, for instance